src/Models/HydrostaticFreeSurfaceModels/split_explicit_free_surface_kernels.jl

```diff
@@ -319,10 +327,15 @@ function iterate_split_explicit!(free_surface, grid, Δt)
     settings  = free_surface.settings
     g         = free_surface.gravitational_acceleration
 
-    Δτ = settings.Δτ * Δt  # we evolve for two times the Δt 
+    preliminary_substeps = calculate_substeps(settings, Δt)
```

Comment:
```suggestion
    N_initial_substeps = calculate_substeps(settings, Δt)
```

using "N" helps remind us that this is a number, like Nx or Nt.